test-apps/ui-test-app/src/frontend/appui/frontstages/component-examples/ComponentExamplesProvider.tsx

```diff
@@ -895,20 +895,21 @@ export class ComponentExamplesProvider {
   };
 
   private static get messageSamples(): ComponentExampleCategory {
+    MessageManager.registerAnimateOutRef(null);
     return {
       title: "Messages",
       examples: [
         createComponentExample("Toast", undefined,
           <UnderlinedButton onActivate={
-            () => IModelApp.notifications.outputMessage(new NotifyMessageDetails(OutputMessagePriority.Info, "This is an info message", undefined, OutputMessageType.Toast))
+            () => MessageManager.displayMessage(new NotifyMessageDetails(OutputMessagePriority.Info, "This is an info message", undefined, OutputMessageType.Toast), undefined, {placement: "top"})
```

Comment:
We should not change the tests to bypass the NotificationManager, since that's what core/frontend uses for messaging.

Reply:
oh.. ok. Reverted those.